src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -271,8 +261,8 @@ private static IEnumerable<string> ProgramPaths(string directory, string[] suffi
 
             var files = new List<string>();
             var folderQueue = new Queue<string>();
-            folderQueue.Enqueue(directory);
-            
```

Comment:
[nit] Can you please update the file to remove changes like adding a line/space which might be adding to git history?

Reply:
I tried to remove this change by changing the line endings for this file. However, this change that you see if because Visual studio changing the line endings from LF to CR LF. When I set it back to LF, the other lines of this file show up as modified. I'll keep this in mind going forward to turn off the automatic line ending fixes when there are inconsistent line endings, however it just made it consistent with the rest of the file (I think it would be good to keep it that way). I don't see any way to fix this other than to re-do everything. Pls let me know if you have any issues with this or with anything else in the PR. Otherwise I shall go ahead and merge it.

Reply:
I tried to remove this change by changing the line endings for this file. However, this change that you see if because Visual studio changing the line endings from LF to CR LF. When I set it back to LF, the other lines of this file show up as modified. I'll keep this in mind going forward to turn off the automatic line ending fixes when there are inconsistent line endings, however it just made it consistent with the rest of the file (I think it would be good to keep it that way). I don't see any way to fix this other than to re-do everything. Pls let me know if you have any issues with this or with anything else in the PR. Otherwise I shall go ahead and merge it.

Reply:
Please go ahead and merge this PR. It was a small nit.